string frame_id
bool run
---
bool result
string message